src/IConfigOptions.ts

```diff
@@ -183,6 +183,12 @@ export interface IConfigOptions {
         // length per voice chunk in seconds
         chunk_length?: number;
     };
+
+    user_notice?: {
```

Comment:
Perhaps it should be an array?

Reply:
I don't think I'd want any user to be tormented that much :P